test/e2e/specs/embedding.test.js

```diff
@@ -0,0 +1,130 @@
+/**
```

Comment:
The "blocks" folder didn't exist at the time these tests were introduced, but at this point this file would be better placed as `specs/blocks/embed.test.js`.